### DISCUSSION TITLE
Support umbracoRedirect in Content Delivery API

### DISCUSSION THREAD

**Discussion by cf-marc** [umbraco/Umbraco-CMS#14544] [Features and ideas]
Hello

In one of our projects we have some kind of "container" without any layout. We tried to use the umbracoRedirect property to redirect to another node, if someone try to access the item with the url. If we send a request to this "container", we receive an 200 OK Status and the umbracoRedirect property is null. It would be nice if we got a 301 Status and the new route to handle the redirect in our frontend correctly.

**Comment by bergmania** [comment]
Hi @cf-marc.

This feature already is in our backlog, but currently we do not have a target release date.
